src/main/kotlin/eda/shoppingBasket/service/dto/ShoppingBasketDTO.kt

```diff
@@ -0,0 +1,13 @@
+package eda.shoppingBasket.service.dto
+
+import java.util.UUID
+
+data class ShoppingBasketDTO(
+    val id: UUID? = null, //we might need this?
```

Comment:
Yes, making this nullable is the right call IMO, you can also omit fields when creating data classes via constuctor i think.